src/stream-ciphers.rst

```diff
@@ -9,18 +9,18 @@ Description
 ~~~~~~~~~~~
 
 A :term:`stream cipher` is a :term:`symmetric-key encryption` algorithm that encrypts a
-stream of bits. Ideally, that stream could be as long as we'd like;
-real-world :term:`stream cipher`\s have limits, but they are normally
-sufficiently large that they don't pose a practical problem.
+stream of bits. Ideally, the stream should be as long as we like, but
+real-world :term:`stream cipher`\s have limits. Though the stream is
+sufficiently large that it does not pose practical problems.
```

Comment:
I think this changes the meaning of the sentence. I'm not saying "it'd be great if we could get a stream of the appropriate size" because appropriate size is specific to the application of a stream cipher, i.e. the specific message you're encrypting. I'm saying that in the abstract, to evaluate a stream cipher, you'd ideally want it to produce unbounded streams. In practice they're not quite unbounded, but they're typically long enough.